test/sequential/test-child-process-pass-fd.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fork = require('child_process').fork;
+const net = require('net');
+
+const N = 80;
```

Comment:
I understand. That was one of the reasons why initially I was running this test only on `OS X`. I chose 80 because it was the magic number that caused the test to fail in my `OS X` almost 100% of the time. Maybe it's not a bad idea to run the test only in `OS X` as it seems an `OS X` only bug, and the basic functionality the test is checking is already covered by `test-cluster-net-send`.